Project.toml

```
@@ -11,16 +11,17 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 
 [compat]
-LearnBase = "0.2, 0.3"
+LearnBase = "0.2, 0.3, 0.4"
```

MLLabelUtils isn't yet compatible with LearnBase `v"0.4"`, which is a slightly breaking version. Check the CI logs you'll find it's `v"0.3"` that's used.

```suggestion
LearnBase = "0.2, 0.3"
```

---

Oh, thanks, my bad. I saw that tests in the PR https://github.com/JuliaML/MLDataPattern.jl/pull/45 are passing, but did not notice that it's actually not used. Will revert this.